### DISCUSSION TITLE
Can we read maps, collections, skins and scores from the files directly?

### DISCUSSION THREAD

**Discussion by Quicksoapy** [ppy/osu#25280] [Q&A] [answered]
I want to be able to do this to backup my scores, skins, maps and collections.

When researching this i did find this: https://github.com/ppy/osu/wiki/User-file-storage
It says here it was planned to have a useful interface available in 2021, but haven't found anything that fits my needs. 
Does someone know if there's a solution?

**Comment by peppy** [comment, marked as answer]
To backup, please backup the full osu! directory.
